### PR TITLE
Create header files for game commands

### PR DESCRIPTION
**Pull request by ilmoro93**
This PR refers to issue https://github.com/OpenLoco/OpenLoco/issues/2069.

Does it make sense to move structs to header files for commands not yet implemented?

### CODE REVIEW

**Comment by AaronVanGeffen** [comment]
Honestly, the thinking at this point was indeed that we'd move them as we implement them. I wouldn't mind accepting your PR, though. Tagging @duncanspumpkin as he may have some thoughts on this as well.

**Comment by duncanspumpkin** [comment]
I've no qualms about doing this separately. It all has to be done eventually so if you want to do it now, by all means go ahead.

**Comment by ilmoro93** [comment]
Ok then, I will move the rest of the structs to new files.

**Comment by AaronVanGeffen** [comment]
Looks like you've already tackled all of the remaining commands, save for the ones that we hadn't created a command struct for yet. Nice work!

**Comment by ilmoro93** [comment]
Yes, there are no more named command struct in **GameCommands.h**. I don't know if anything else can be done at this point.

**Comment by AaronVanGeffen** [comment]
We'll have to make named structs for the others eventually, but that's really something for another PR.

File: src/OpenLoco/src/GameCommands/Buildings/RemoveBuilding.h [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include "GameCommands/Buildings/CreateBuilding.h"
```
Comment (AaronVanGeffen) [inline src/OpenLoco/src/GameCommands/Buildings/RemoveBuilding.h:3]: This seems like an unnecessary include. Could it be removed?

Comment (AaronVanGeffen) [reply src/OpenLoco/src/GameCommands/Buildings/RemoveBuilding.h:3]: I see it was added due to the `BuildingPlacementArgs`-based constructor. Never mind then :)

**Review by AaronVanGeffen — APPROVED** [review]

**Comment by AaronVanGeffen** [comment]
Looks good to me. Testing revealed no abnormalities either. Nice work!

**Pull request merged**